### PR TITLE
remove join pages from membership

### PR DESCRIPTION
**Pull request by johnduffell**
## Why are you doing this?

Membership acquisition is not needed any more.  We prefer people to contribute or subscribe.

### CODE REVIEW

**Comment by rupertbates** [comment]
I think we should probably redirect the `choose-tier` url and the various `tier/change` urls

File: frontend/conf/routes [hunk 1]
```diff
@@ -20,6 +20,8 @@ GET         /feature/:feature                          controllers.FeatureOptIn.
 GET         /feature/:feature/:onOrOff                 controllers.FeatureOptIn.setFeatureOnOrOff(feature: Feature, onOrOff: OnOrOff)
 
 # Join
+GET         /join/*any                                controllers.Redirects.whySupportRedirectIgnore(any)
```
Comment (rupertbates) [inline frontend/conf/routes:23]: We could also disallow `/join` in robots.txt

Comment (johnduffell) [reply frontend/conf/routes:23]: Hadn't thought about that but what benefit are you expecting from robots.txt?
I assume search engines will follow the redirect, ideally we want people who google join membership to get to the support page.  I did a small amount of research just now and they seem to just say use a redirect.
 https://support.google.com/webmasters/answer/93633

Comment (rupertbates) [reply frontend/conf/routes:23]: That's a fair point, I always find that it's quite hard to get Google to remove paths from the index but as we're redirecting that should be fine. 👍

File: frontend/app/model/Faq.scala [hunk 2]
```diff
@@ -38,41 +38,6 @@ object Faq {
   )
 
   val help = List(
-    Section("Guardian Membership: the basics", List(
```
Comment (johnduffell) [inline frontend/app/model/Faq.scala:41]: image

**Pull request merged**